manifests/role/webhost/www_lib_vm.pp

```diff
@@ -35,9 +35,12 @@
   include nebula::profile::www_lib::dependencies
   include nebula::profile::www_lib::perl
   include nebula::profile::www_lib::php
-  #  include nebula::profile::www_lib::shibboleth
   include nebula::profile::www_lib::apache
   include nebula::profile::unison
 
+  class { 'nebula::profile::shibboleth':
+    config_source => 'puppet:///shibboleth-www_lib'
```

Comment:
`not-a_fan` But otherwise looks good to me.

Reply:
Heh! We'll get that later. Thanks!